save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -449,10 +489,43 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
         }
     }
 
+    private fun setNewDescription(value: String) {
+        descriptionTmp = value
+    }
+
+    private fun onOrganizationSave() {
```

Comment:
This function should do nothing if `state.organization == null`, but to achieve that you are currently doing two checks and launching a coroutine even if the value is null. Why not accept `Organization` as a non-nullable parameter in this function and perform nullability check before calling this function?

Reply:
Fixed